packages/core/src/language-server/messages.cts

```diff
@@ -7,10 +7,15 @@ export type Request<Name extends string, T> = {
 
 export const GetIRRequest = makeRequestType(
   'glint/getIR',
-  ProtocolRequestType<GetIRParams, GetIRResult | null, void, void, void>
+  ProtocolRequestType<RequestParams, GetIRResult | null, void, void, void>
 );
 
-export interface GetIRParams {
+export const SortImportsRequest = makeRequestType(
+  'glint/sortImports',
+  ProtocolRequestType<RequestParams, TextEdit[] | null, void, void, void>
+);
+
+export interface RequestParams {
```

Comment:
It's arguably incidental that `getIR` and `sortImports` both just want a `uri` param, while other requests might require more (or less!) information. Given that, let's keep separate `xParams`/`xResult` types for each message.